src/lib/Form/Factory/FormFactory.php

```diff
@@ -121,6 +123,28 @@ public function createContentDraft(
     }
 
     /**
+     * @param ContentUpdateMainLocationData|null $data
+     * @param string|null $name
+     *
+     * @return FormInterface
+     *
+     * @throws InvalidOptionsException
+     */
+    public function updateContentMainLocation(
```

Comment:
Code placement isn't logical to me, there are methods like `removeLocation()`, `swapLocation()`, somewhere around that area seems like a better fit.